ui/apps/platform/src/Containers/VulnMgmt/RiskAcceptance/FalsePositiveCVEs/FalsePositiveCVEsTable.tsx

```diff
@@ -237,15 +242,21 @@ function FalsePositiveCVEsTable({
                                         </Button>
                                     </Td>
                                     <Td dataLabel="Comments">
-                                        <RequestCommentsButton
-                                            comments={row.vulnerabilityRequest.comments}
-                                            cve={row.vulnerabilityRequest.cves.cves[0]}
-                                        />
+                                        {row.vulnerabilityRequest ? (
+                                            <RequestCommentsButton
+                                                comments={row.vulnerabilityRequest.comments}
+                                                cve={row.vulnerabilityRequest.cves.cves[0]}
+                                            />
+                                        ) : (
+                                            'N/A'
```

Comment:
An observation, not a change request. to distinguish **Not applicable** and **Not available** for new work